src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private BetDao betDao = new BetDaoImpl();
+    private UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                scanner.close();
+                return;
+            }
+            Bet bet = null;
+            User user = null;
+            try {
+                String[] userAndBetData = command.split(" ");
+                String name = userAndBetData[0];
+                int age = Integer.parseInt(userAndBetData[1]);
+                if (age < 18) {
+                    throw new IllegalArgumentException();
```

Comment:
let's maybe just sout the message here and continue with the next iteration? what is the point of throwing and then catching this exception?

Reply:
Agree. Done.